### PR TITLE
Reorg bug fix

### PR DESCRIPTION
**Pull request by atif-konasl**
**What type of PR is this?**
> Bug fix

**What does this PR do? Why is it needed?**
This PR will fix bug of current l15 reorg.

**Which issues(s) does this PR fix?**

Fixes #

**Other notes for review**

### CODE REVIEW

**Review by blazejkrzak — CHANGES_REQUESTED** [review]
we miss tests

**Comment by atif-konasl** [comment]
> we miss tests

Yeah you right. It's not a massive change so we can test for these simple getter and setter method `setLatestSentEpoch` and `getLatestSentEpoch` later.

**Comment by blazejkrzak** [comment]
> > we miss tests
> 
> Yeah you right. It's not a massive change so we can test for these simple getter and setter method `setLatestSentEpoch` and `getLatestSentEpoch` later.

what does block us to write meaningful tests now?

**Comment by rryter** [comment]
I agree, let's add the tests now.

File: proto/eth/v1alpha1/beacon_chain.proto [hunk 1]
```diff
@@ -919,6 +919,9 @@ message Reorg {
     bytes van_parent_hash = 1;
     // pan_parent_hash is the hash of pandora sharding block of the parent block re-org new block
     bytes pan_parent_hash = 2;
+    // new_slot is the new head slot after re-org
+    uint64 new_slot = 3 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/eth2-types.Slot"];
+
```
Comment (blazejkrzak) [inline proto/eth/v1alpha1/beacon_chain.proto:924]: redundant newline

**Comment by atif-konasl** [comment]
> > > we miss tests
> > 
> > 
> > Yeah you right. It's not a massive change so we can test for these simple getter and setter method `setLatestSentEpoch` and `getLatestSentEpoch` later.
> 
> what does block us to write meaningful tests now?

Time is blocking us. I am really bz with testing clients in our vms. Could you write some unit test cases?

File: proto/eth/v1alpha1/beacon_chain.proto (hunk 1 above)
Comment (atif-konasl) [reply proto/eth/v1alpha1/beacon_chain.proto:924]: Removing

Comment (mxmar) [reply proto/eth/v1alpha1/beacon_chain.proto:924]: After remove please close/solve this conversation :)

**Review by mxmar — CHANGES_REQUESTED** [review]
Implementation in overall look ok to me but I agree that we should provide component test on top of it to avoid next network stall and confirm fix. Anyway I think we can build/release binary of it and provide to community who faced network stall.

**Comment by mxmar** [comment]
I'm working on `vanguard` side trying to provide proper test scenario on top of this PR changes: https://github.com/lukso-network/vanguard-consensus-engine/pull/74

**Pull request merged**